src/lib/abrt_types.c

```diff
@@ -82,7 +81,7 @@ int try_get_map_string_item_as_bool(map_string_t *ms, const char *key, int *valu
     GET_ITEM_OR_RETURN(option, ms, key);
 
     *value = string_to_bool(option);
-    return true;
+    return 1;
```

Comment:
Unrelated change. Is this causing warnings?

Reply:
It was just the inconsistency bugging me. There are 0s and 1s everywhere and bam, a `true` here.

Reply:
Aha.

I would at least make this a separate commit. What do you think about changing it all to C99 booleans? Not as much mental overhead in trying to enumerate the possible return values with those.

Reply:
OK, I'll perhaps leave that to later.